README.md

```diff
@@ -79,9 +79,26 @@ Some secrets and sensitive configuration files are needed to properly operate ce
    ...
    ```
 
+1. Add a `cypress.env.json` in the `apps/mull-ui-e2e/` project folder. See `docs-and-resources` channel on slack for our secret key.
+
+   ```properties
+   {
+      "ACCESS_TOKEN_SECRET": "<secret-key>"
+   }
+   ```
+
+1. Add a `host.env.ts` under the `libs/types/src/` folder. This file specify the hostnames where the application are currently being hosted, and makes it easy to change them.
+
+   ```ts
+   export const UI_HOSTNAME = 'hostname';
+   export const API_HOSTNAME = 'hostname';
+   ```
+
 1. Start the server and client dev server in another terminal to test that everything is working correctly.
    - `nx serve mull-api`
-   - `nx serve mull`
+   - `nx serve mull-ui`
+   - If you specified a host other than `localhost` in your `host.env.ts`, then make sure to add the `--host` flag so that the webserver can be configured properly to serve the app on the right host
+     - example: `nx sere mull-ui --host 123.123.123.123`
```

Comment:
```suggestion
     - example: `nx serve mull-ui --host 123.123.123.123`
```

Reply:
thanks for looking out! Sorry for the sloppy readme updates on my end 😅